### ISSUE TITLE
trying to use different domain on AMI of my peertube instance

### ISSUE THREAD

**Issue by OceanTechie** [Chocobozzz/PeerTube#2571]
i made an image of my aws ec2 ubuntu instance with peertube installed against my domain name such as www.myexampleurl.com 

i launched a new ec2 instance based on my AMI (backup of above) - and i have another domain name to use on that server instance - say, www.myexampleurlDEV.com

Peertube is starting and log shows it is still referring to www.myexampleurl.com - even though I modified production.yaml, nginx sites enabled file.  I note installation docs state domain can never be changed on peertube once initiated/started

so i removed peertube following a past issue response instructions (remove node_modules folder and re-run yarn build per installation instructions).   my nginx sites-enabled has my new domain as does production.yaml but no go on starting peertube.  Looks like the method of remove (rm node_modules and redo yarn) was not sufficient a method to rid the domain embedded somehow in peertube as the journalctl -feu output is showing use of the original domain url

any way around this or i must start from a raw ec2 instance and redo all installation steps in order to use a different domain for my new ec2 instance (prerequisites took quite a lot of things to resolve along the way)

I do note the dev branch method and creating plugins as means to modify client UI and behaviors, but i'm trying to inject script into video-watch.component.html to use a video-js plugin as my player (which i notice i only see when I start to follow CLI tools installation and unzip the git zip file downloaded in that)

Also, I notice admin, advanced configuration - allows for adding javascript but i don't see how to target an individual client html page.

version 2.1.1 of peertube on ubuntu linux 18.04 and nodejs 10.13

**Comment by Chocobozzz** [comment]
If you don't want to keep data of your old peertube instance, just reinstall peertube.
If you want to keep your old data, you can try https://docs.joinpeertube.org/#/maintain-tools?id=update-hostjs

**Comment by OceanTechie** [comment]
I had tried full uninstall and re-install - for uninstall, i followed a response to issue here advice and deleted node_modules and re-ran the yarn build.  That did not impact the domain used - still the old one and my production.yaml, nginx/sites-enabled both refer to new domain.

So i tried the CLI tools server option to update domain.  It does not prompt for the new domain and the output is listing our the old domain name.  Is it reading that in from one of the configs?  Which one?   Further it says npm update check failed and needs to chmod access to /home/ubuntu/.config folder.

Finally, doing sudo systemctl start peertube - it hangs - does not go back to prompt. If i ctrl-c and and check journalctl -feu peertube, no new log content is seen.

here is the update command I ran.
  
sudo -u peertube NODE_CONFIG_DIR=/var/www/peertube/config NODE_ENV=production npm run update-host
